Modules/Clients/Feature/Modules/RelationsExportImportTest.php

```
@@ -0,0 +1,118 @@
+        foreach ($relations as $relation) {
+            $this->assertStringContainsString($relation->name, $content);
+        }
+    }
+
+    #[Test]
+    #[Group('export')]
+    public function it_exports_relations_downloads_excel_with_correct_data(): void
+    {
+        $this->markTestIncomplete();
+        /* Arrange */
+        $relations = Relation::factory()->for($this->company)->count(3)->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$relations'. (undefined)

(UnusedLocalVariable)